health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/CustomBarChart.js

```
@@ -0,0 +1,181 @@
+    key: id,
+    type: "metric",
+    tenantId,
+    requestDate: value?.requestDate != null ? { ...value?.requestDate, startDate: value?.range?.startDate?.getTime(), endDate: value?.range?.endDate?.getTime() } : requestDate,
+    filters: {...value?.filters, projectTypeId: selectedProjectTypeId},
+    moduleLevel: value?.moduleLevel
+  });
+  const chartData = useMemo(() => {
+    if (!response) return null;
+    setChartDenomination(response?.responseData?.data?.[0]?.headerSymbol);
+    const dd = response?.responseData?.data?.map((bar) => {
+      let plotValue = bar?.plots?.[0].value || 0;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'plotValue' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)